### DISCUSSION TITLE
add/move to the side, right and left navigation button

### DISCUSSION THREAD

**Discussion by marcarc** [fengyuanchen/viewerjs#460] [Q&A]
Hi!
I was wondering, if there is an option, to add, or move the navigation (right & left) buttons on the side, instead of at the button in the toolbar, like that:

![button](https://user-images.githubusercontent.com/63582018/113890621-9e4a8480-97c4-11eb-8b9e-90e05c76bc78.png)

Or should I just override the "toolbar" CSS and move the buttons? Or any better ideas?

Thank you for your reply :)

**Comment by fengyuanchen** [comment]
See #461.
